src/jit/codegenarmarch.cpp

```diff
@@ -1617,11 +1766,29 @@ void CodeGen::genCallInstruction(GenTreeCall* call)
         }
         else
         {
-            regNumber argReg = curArgTabEntry->regNum;
             genConsumeReg(argNode);
```

Comment:
I updated `genUnspillRegIfNeeded` (https://github.com/hseok-oh/coreclr/blob/b5d86a6cec83522e6dcb784076f9136d81547129/src/jit/codegenlinear.cpp#L1007-L1035) called by `genConsumeReg`. 
And I added `gtSpillFlags` field in `GenTreePutArgSplit`. (https://github.com/hseok-oh/coreclr/blob/b5d86a6cec83522e6dcb784076f9136d81547129/src/jit/gentree.h#L4969)
To handle this flag, I also updated `genProduceReg` in `codegenlinear.cpp`, `rsSpillTree` and `rsUnspillInPlace` in `regset.cpp`. Is there more function to update to consume all registers?